app/views/jobseekers/application/_account_header.html.haml

```diff
@@ -0,0 +1 @@
+= render(Shared::DashboardComponent.new(heading: current_jobseeker.email, navigation_items: [account_navigation_link(t("jobseekers.saved_jobs.index.page_title"), jobseekers_saved_jobs_path), account_navigation_link(t("jobseekers.subscriptions.index.page_title"), jobseekers_subscriptions_path), account_navigation_link(t("jobseekers.accounts.show.page_title"), jobseekers_account_path)], link: false))
```

Comment:
You don't need to pass in `link: false`, you can just omit it, and then when you do `if @link` maybe do something like `if link.present`